src/components/ExecuteButton.js

```diff
@@ -74,12 +74,15 @@ export class ExecuteButton extends React.Component {
       <path d="M 11 9 L 24 16 L 11 23 z" />;
 
     return (
-      <div className="execute-button-wrap">
+      <div
+        className="execute-button-wrap"
+        tabIndex="-1">
```

Comment:
I put that there because since it's a surrounding div I think it's supposed to be marked as non tabbable with a -1? I may be totally wrong though.